tests/rptest/tests/datalake/datalake_services.py

```diff
@@ -211,7 +213,7 @@ def wait_for_translation_until_offset(self,
                                           topic,
                                           offset,
                                           partition=0,
-                                          timeout=30,
+                                          timeout=120,
```

Comment:
nit: how about just setting the cluster property wherever we set the commit interval? Waiting for longer (and this _is_ quite a while) seems like it leaves room for undesirable scheduling bugs to be ignored

Reply:
Sure, same feedback for the rest of the ducktape changes. Why not just have them set the cluster properties? Every test basically expects that anyway by virtue of the commit interval being so low

Reply:
Regarding rest of the changes

> tests/rptest/tests/datalake/recovery_mode_test.py

This is a special test that is running into some flakiness in coordinator API post restore and hitting backoff limits (hence a special bump in timeout)

The other changes could be removed too, probably (most of them were fiddling with timeouts to see what works and what doesn't. I'll merge this so Michal can rebase, I have another PR coming up soon, will cleanup test code a bit more.